modules/mobile-agents/windows/jax-rs/src/main/java/org/wso2/carbon/mdm/mobileservices/windows/common/util/UsernameTokenValidator.java

```diff
@@ -37,92 +37,94 @@
+    private static final int DOMAIN_SEGMENT = 1;
+    private static final String DELIMITER = "@";
+    private static Log log = LogFactory.getLog(UsernameTokenValidator.class);
 
-	/**
-	 * This method validates the username token in SOAP message coming from the device.
-	 * @param credential  - Username token credentials coming from device
-	 * @param requestData - Request data associated with the request
-	 * @return - Credential object if authentication is success, or null if not success
-	 * @throws WSSecurityException
-	 */
-	@Override
-	public Credential validate(Credential credential, RequestData requestData) throws
-			                   WSSecurityException {
+    /**
+     * This method validates the username token in SOAP message coming from the device.
+     *
+     * @param credential  - Username token credentials coming from device
+     * @param requestData - Request data associated with the request
+     * @return - Credential object if authentication is success, or null if not success
+     * @throws WSSecurityException
+     */
+    @Override
+    public Credential validate(Credential credential, RequestData requestData) throws
+            WSSecurityException {
 
-		String domainUser = credential.getUsernametoken().getName();
-		String[] domainUserArray = domainUser.split(DELIMITER);
-		Credential returnCredentials;
-		String user = domainUserArray[USER_SEGMENT];
-		String domain = domainUserArray[DOMAIN_SEGMENT];
-		String password = credential.getUsernametoken().getPassword();
+        String domainUser = credential.getUsernametoken().getName();
+        String[] domainUserArray = domainUser.split(DELIMITER);
+        Credential returnCredentials;
+        String user = domainUserArray[USER_SEGMENT];
+        String domain = domainUserArray[DOMAIN_SEGMENT];
+        String password = credential.getUsernametoken().getPassword();
 
-		try {
-			if (authenticate(user, password, domain)) {
-				returnCredentials = credential;
-			} else {
-				String msg = "Authentication failure due to incorrect credentials.";
-				log.error(msg);
-				throw new WindowsDeviceEnrolmentException(msg);
-			}
-			//Generic exception is caught here as there is no need of taking different actions for
-			//different exceptions.
-		} catch (Exception e) {
-			String msg = "Failure occurred in the credential validator.";
-			log.error(msg, e);
-			throw new WSSecurityException(msg, e);
-		}
-		return returnCredentials;
-	}
+        try {
+            if (authenticate(user, password, domain)) {
+                returnCredentials = credential;
+            } else {
+                String msg = "Authentication failure due to incorrect credentials.";
+                log.error(msg);
+                throw new WindowsDeviceEnrolmentException(msg);
+            }
+            //Generic exception is caught here as there is no need of taking different actions for
+            //different exceptions.
+        } catch (Exception e) {
```

Comment:
As a best practice, use multi-catch statements in place of catching generic exceptions.